src/test/java/uk/gov/pay/products/client/publicapi/PublicApiRestClientTest.java

```diff
@@ -19,25 +22,36 @@
 import static junit.framework.TestCase.assertTrue;
 import static org.apache.http.HttpStatus.SC_BAD_REQUEST;
 import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
 import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertThat;
 import static org.junit.Assert.fail;
 import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 import static uk.gov.pay.commons.model.Source.CARD_PAYMENT_LINK;
 import static uk.gov.pay.products.matchers.PaymentResponseMatcher.hasAllPaymentProperties;
 import static uk.gov.pay.products.stubs.publicapi.PublicApiStub.createPaymentRequestPayload;
 import static uk.gov.pay.products.stubs.publicapi.PublicApiStub.setupResponseToCreatePaymentRequest;
 import static uk.gov.pay.products.stubs.publicapi.PublicApiStub.setupResponseToGetPaymentRequest;
 
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
I can't see any new `@Mock` annotations in the test, do we need this `@RunWith`?

Reply:
Thanks for spotting it. I have dropped it. I used `@Mock` annotation initially and then later decided against it because it didn't work with the static variable `RestClientConfiguration`.